tests/cases/issues/test_issue_258.py

```diff
@@ -0,0 +1,62 @@
+#  Authors: Eddie Bergmane <eddiebergmanehs@gmail.com>
+#            + All contributors to <https://github.com/smarie/python-pyfields>
+#
+#  License: 3-clause BSD, <https://github.com/smarie/python-pyfields/blob/master/LICENSE>
+#
```

Comment:
We do not insert copyright headers in tests anymore - this comes from a stupid copy-paste from a different repository, I removed it from the other test files. Thanks for pointing it :)

```suggestion
```

Reply:
Ahh, I just copied and pasted a random test file, I'll remove it :)